### PR TITLE
Fix GZip/Brotli compression for webgl

### PR DESCRIPTION
**Pull request by mob-sakai**
#### Changes

See #75

- Add GZip/Brotli compression tests for webgl module
- Support GZip compression for webgl (for 2020.x, 2020.2.0 and 2020.2.1)
  - Add -f as default option for gzip
- Support Brotli compression for webgl (for 2018.x, 2019.x and 2020.1.x)
  - Install `python-setuptools`
  - Copy brotli egg for linux

```
REPOSITORY       TAG                   IMAGE ID       CREATED          SIZE
unityci/editor   dev                   ed64129acfe7   49 seconds ago   6.21GB
unityci/editor   2020.1.11f1-webgl-0   e10428a15532   5 days ago       6.2GB
```

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)

#### Test

- Before modification: https://github.com/mob-sakai/docker/actions/runs/565959178
  - NOTE: In Unity 2018.4.15 or earlier, even if brotli compression fails, the build will be marked as successful.
- After modification: https://github.com/game-ci/docker/actions/runs/566007780

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -224,6 +224,40 @@ jobs:
           targetPlatform: ${{ matrix.platform }}
           allowDirtyBuild: true
 
+      ###########################
+      #       Build (GZip)      #
+      ###########################
+      - name: "⚙️ Setup (GZip)"
+        if: env.MODULE == 'webgl'
+        run: |
+          sed -i -e 's/webGLCompressionFormat: .*$/webGLCompressionFormat: 1/' reference-project-test/ProjectSettings/ProjectSettings.asset
```
Comment (webbertakken) [inline .github/workflows/test.yml:233]: Could you add a comment in here, saying what this does? For example: "Set WebGL compression format to gzip".

**Comment by webbertakken** [comment]
Great! Thanks for referencing the relevant sources.

**Comment by webbertakken** [comment]
Looks complete, is this still a "work in progress"? I will merge it after you mark it ready for review.

**Comment by mob-sakai** [comment]
@webbertakken 
OK, it's ready for review.

**Pull request merged**